pkg/odo/cli/deploy/deploy.go

```diff
@@ -19,6 +21,7 @@ import (
 	odoutil "github.com/redhat-developer/odo/pkg/odo/util"
 	"github.com/redhat-developer/odo/pkg/version"
 
+	_component "github.com/redhat-developer/odo/pkg/component"
```

Comment:
let's give it a more descriptive name  `_` looks weird. 
How about `odocomponent`?